src/script/components/MessagesList/Message/ContentMessage/asset/AudioAsset.tsx

```diff
@@ -75,14 +75,27 @@ const AudioAsset: React.FC<AudioAssetProps> = ({
       try {
         const blob = await loadAsset(asset.original_resource());
         setAudioSrc(window.URL.createObjectURL(blob));
-        audioElement?.play();
       } catch (error) {
         logger.error('Failed to load audio asset ', error);
       }
       asset.status(AssetTransferState.UPLOADED);
     }
   };
 
+  useEffect(() => {
+    if (audioSrc && audioElement) {
+      const playPromise = audioElement.play();
+
+      if (playPromise !== undefined) {
+        playPromise.catch(error => {
```

Comment:
```suggestion
        playPromise?.catch(error => {
```
I suggest using optional chaining so you can get rid of `if` statement

Reply:
Fixed